jax/experimental/mosaic/gpu/fragmented_array.py

```diff
@@ -46,12 +46,11 @@ class WGSplatFragLayout:
   wants. This means we can trivially broadcast, reshape and do
   elementwise operations with all other layouts.
 
-  Example:
+  Examples:
 
-  To load a value in
-  ```
-  FragmentedArray.splat(memref.load(ref_1d, [1]), (10,20,2))
-  ```
+  To load a value in:
```

Comment:
Does this need to be indented?

Reply:
I think probably yes, but I've actually just reverted that change to keep this PR self-contained. I have a (potentially bad) plan for the other lint!